src/lib/es5.d.ts

```diff
@@ -210,6 +210,8 @@ interface ObjectConstructor {
       * Returns the names of the enumerable properties and methods of an object.
       * @param o Object that contains the properties and methods. This can be an object that you created or an existing Document Object Model (DOM) object.
       */
+    keys<T>(o: Array<T>): string[];
+    keys<T extends { [key: string]: any }>(o: T): (keyof T & string)[];

```

Comment:
i would rather leave it `string[]` then. adding the `& string` removes any type safety the literal types give you.